### PR TITLE
Issues batch3

### PR DESCRIPTION
**Pull request by ana-cc**
Only 4 issues in this batch - #15, #20, #21, #22

### CODE REVIEW

File: draft-ietf-tsvwg-dscp-considerations-04.xml [hunk 1]
```diff
@@ -733,19 +743,23 @@ A DSCP can sometimes referred to by name, such as "CS1", and
...
-          interconnected networks. Use of the MPLS Short Pipe Model favours
-          remarking unexpected DSCP values to zero in the absence of an
-          additional agreement(s), as explained in <xref
-          target="RFC8100">RFC8100</xref>. This can result in bleaching
-          (/Bleach/).</t>
+	  <t>RFC8100 recommends preserving the notion of end-to-end service
+          classes, and recommends a set of standard DSCPs mapped to a small set of
+         standard PHBs at interconnection.  The key requirement is that the DSCP at the
+         network ingress is restored at the network egress.  The current version of
+         RFC8100 limits the number DSCPs to 6 and 3 more are suggested for extension.
```
Comment (gorryfair) [inline draft-ietf-tsvwg-dscp-considerations-04.xml:750]: insert /of/ before /DSCPs/

File: draft-ietf-tsvwg-dscp-considerations-04.xml [hunk 2]
```diff
@@ -796,14 +810,9 @@ A DSCP can sometimes referred to by name, such as "CS1", and
...
-        provides a mobile ISP to ISP QoS mapping mechanism, and
-        interconnection with other IP networks in the general Internet. This
-        describes the case where the DSCP marking from a Service Provider
-        cannot be trusted (depending on the agreement between the Service
-        Provider and its IPX Provider), allowing the IPX Provider to remark
-        the DSCP value to a static default value.</t>
+        <xref target="GSMA-IR-34">GSMA IR.34</xref>. 
+This was previously specified as the Inter-Service Provider IP Backbone Guidelines, and provides a mobile ISP to ISP QoS mapping mechanism, and interconnection with other IP networks in the general Internet. If realized by an IP VPN, the operator is free to apply its DS Domain internal codepoint scheme at outer headers and inner IPX DSCPs may be transported transparently. The guidelines also describe a case where the DSCP marking from a Service Provider cannot be trusted (depending on the agreement between the Service Provider and its IPX Provider), in which situation the IPX Provider can remark the DSCP value to a static default value.
+        </t>
```
Comment (gorryfair) [inline draft-ietf-tsvwg-dscp-considerations-04.xml:815]: Are we using UK English or US English? /realized/realised/ ?

**Review by gorryfair — APPROVED** [review]

File: draft-ietf-tsvwg-dscp-considerations-04.xml (hunk 2 above)
Comment (ana-cc) [reply draft-ietf-tsvwg-dscp-considerations-04.xml:815]: fixed

File: draft-ietf-tsvwg-dscp-considerations-04.xml (hunk 1 above)
Comment (ana-cc) [reply draft-ietf-tsvwg-dscp-considerations-04.xml]: fixed

**Pull request merged**